lib/mocha.js

```diff
@@ -12,6 +12,8 @@ var escapeRe = require('escape-string-regexp');
 var path = require('path');
 var reporters = require('./reporters');
 var utils = require('./utils');
+var Random = require('random-js');
+var MAX_SAFE_INTEGER = 0x1fffffffffffff;
 
```

Comment:
Yes, sorry.  `0x1fffffffffffff` is the value of `Number.MAX_SAFE_INTEGER` *if* `Number.MAX_SAFE_INTEGER` is implemented.  But we're not guaranteed that, of course, and I didn't want to pull in some polyfill.  I'll add a comment.